helm/app-operator-chart/templates/rbac.yaml

```diff
@@ -72,11 +72,12 @@ rules:
   verbs:
   - "create"
 - apiGroups:
-  - extensions
+  - apps
   resources:
   - deployments
   verbs:
-  - 'create'
+  - "create"
```

Comment:
Do we need the create? 

I think that was for tiller and I forgot to remove it. 🤦‍♂

Reply:
Oh, then we can delete it!